pyocd/probe/pydapaccess/interface/__init__.py

```diff
@@ -59,6 +59,6 @@
     elif system == "Linux":
         USB_BACKEND = "pyusb"
     else:
-        raise DAPAccessIntf.DeviceError("No USB backend found")
+        raise DAPAccessIntf.DeviceError("Unknown platform! No USB Interface backend autoselected.")
```

Comment:
Please change "Interface" to "interface" (not capitalised), and "autoselected" to "auto-selected" (the dictionary says the version without a dash is not a valid word).